index.js

```diff
@@ -58,13 +59,16 @@ var templates = get_templates(p);
  */
 var mailCredentials = {
   port: process.env.WATCHMEN_AUTH_NODEMAILER_PORT,
-  host: process.env.WATCHMEN_AUTH_NODEMAILER_HOST,
-  auth: {
-    user: process.env.WATCHMEN_AUTH_NODEMAILER_USER,
-    pass: process.env.WATCHMEN_AUTH_NODEMAILER_PASS
-  }
+  host: process.env.WATCHMEN_AUTH_NODEMAILER_HOST
 };
 
+if (process.env.WATCHMENT_AUTH_NODEMAILER_NO_AUTH != 'true') {
```

Comment:
negative negative positive kinda wrecks your brain trying to brain compile this, possibly inverting it will make things clearer.